pkgs/development/python-modules/jaxlib/prefetch.sh

```diff
@@ -14,7 +14,6 @@ for py in "310" "311" "312" "313"; do
     prefetch "$py" "x86_64-linux" "jaxlib-bin"
     prefetch "$py" "aarch64-linux" "jaxlib-bin"
     prefetch "$py" "aarch64-darwin" "jaxlib-bin"
-    prefetch "$py" "x86_64-darwin" "jaxlib-bin"
```

Comment:
i'm not seeing x86_64-darwin in bin.nix or srcs. i'm assuming we already get rid of it there?